src/components/editMeasure/testCases/components/testCaseLanding/qiCore/CreateCodeCoverageNavTabs.tsx

```diff
@@ -138,68 +138,83 @@ export default function CreateCodeCoverageNavTabs(props: NavTabProps) {
         />
       </Tabs>
       <div tw="flex flex-wrap space-x-4 justify-end h-10">
-        <Button
-          variant="danger-primary"
-          disabled={!canEdit || measure?.testCases?.length === 0}
-          onClick={onDeleteAllTestCases}
-          data-testid="delete-all-test-cases-button"
-        >
-          <KeyboardArrowRightIcon
-            style={{ margin: "0 5px 0 -2px" }}
-            fontSize="small"
-          />
-          Delete All
-        </Button>
-        <Button
-          variant="outline"
-          onClick={onImportTestCases}
-          disabled={!canEdit}
-          data-testid="import-test-cases-button"
-        >
-          <FileUploadIcon style={{ margin: "0 5px 0 -2px" }} fontSize="small" />
-          Import from MADiE
-        </Button>
-        {featureFlags?.qiCoreBonnieTestCases && (
+        {!featureFlags.TestCaseListActionCenter && (
+          //Remove button when feature flag is removed
           <Button
-            onClick={() => {
-              if (onImportTestCasesFromBonnie) {
-                onImportTestCasesFromBonnie();
-              }
-            }}
-            disabled={!canEdit}
-            data-testid="import-test-cases-from-bonnie-button"
+            variant="danger-primary"
+            disabled={!canEdit || measure?.testCases?.length === 0}
+            onClick={onDeleteAllTestCases}
+            data-testid="delete-all-test-cases-button"
           >
-            <FileUploadIcon
+            <KeyboardArrowRightIcon
               style={{ margin: "0 5px 0 -2px" }}
               fontSize="small"
             />
-            Import From Bonnie
+            Delete All
           </Button>
         )}
-        <Button
-          disabled={!canEdit}
-          onClick={createNewTestCase}
-          data-testid="create-new-test-case-button"
-        >
-          <AddIcon style={{ margin: "0 5px 0 -2px" }} fontSize="small" />
-          New Test Case
-        </Button>
+        {((featureFlags.TestCaseListActionCenter && canEdit) ||
+          !featureFlags.TestCaseListActionCenter) && (
```

Comment:
same here

Reply:
AC specifically says otherwise
> The Import buttons and the New Case button should not appear on the screen if the user does own the measure or it's not shared with them OR if it's not in a draft state